client/landing/stepper/declarative-flow/internals/steps-repository/sensei-launch/index.tsx

```diff
@@ -68,11 +33,58 @@ const SenseiLaunch: Step = ( { navigation: { submit } } ) => {
 	} else if ( retries < 0 ) {
 		progress.percentage = 100;
 	}
+	return progress;
+};
+
+const SenseiLaunch: Step = ( { navigation: { submit } } ) => {
+	const [ retries, setRetries ] = useState< number >( 0 );
+	const maxRetries = 40;
+	const progress = useRetriesProgress( retries, maxRetries, 15 );
+
+	const { pollPlugins, isPluginInstalled, queuePlugin } = useAtomicSitePlugins();
+	const additionalPlugins = useMemo( () => getSelectedPlugins( getSelectedPurposes() ), [] );
```

Comment:
Built it this way because we would also need the purposes directly for saving the option, but doesn't really make sense now so I'll just move to a single `getSelectedPlugins`.